src/middlewares/getOneArticle.js

```diff
@@ -0,0 +1,36 @@
+import models from '../api/models';
+import statusCode from '../helpers/constants/status.codes';
+
+const { Article } = models;
+/**
+ * containing all user's model controllers (signup, login)
+ *
+ * @export
+ * @class getOneArticle
+ */
+class getOneArticle {
```

Comment:
@MCFrank16 why do you need a middleware for this? plus the comments in the file are confusing.

Reply:
this middleware helps me to check if the slug in the req.params exists or not, and it can still used by anyone who wants to check on whether the slug exists or not